simphony_mayavi/sources/engine_source.py

```diff
@@ -13,30 +13,7 @@
 
 
 class EngineSource(CUDSSource):
-    """ A mayavi source for reading data from a SimPhoNy Engine
-
-    Attributes
```

Comment:
Attributes should stay in the class docstring only `Parameters` and `Arguments` should be in the `__init__.py`